semgrep/semgrep/semgrep_types.py

```diff
@@ -41,9 +42,17 @@ class OPERATORS:
     OPERATORS.REGEX: ["pattern-regex"],
 }
 
+START_LINE = "__line__"
```

Comment:
Don't these only matter in parsing the yaml? Can this be defined where the custom parser is defined?